src/commands/appdistribution-testers-remove.ts

```diff
@@ -0,0 +1,33 @@
+import {Command} from "../command";
+import * as utils from "../utils";
+import {requireAuth} from "../requireAuth";
+import {FirebaseError} from "../error";
+import {needProjectNumber} from '../projectUtils';
+import {AppDistributionClient} from '../appdistribution/client';
+import {getEmails} from '../testerEmailParser';
+
+
+module.exports = new Command("appdistribution:testers:remove [emails...]")
+    .description("Delete Testers")
```

Comment:
nit: Testers should be lowercase i.e. "Delete testers"

Reply:
Also rename to be "Remove testers" to be more consistent with the command name